pkg/sql/distsqlrun/sorterstrategy.go

```diff
@@ -34,7 +35,7 @@ type sorterStrategy interface {
 	// It returns once either all the input has been exhausted or the consumer
 	// indicated that no more rows are needed. In any case, the caller is
 	// responsible for draining and closing the producer and the consumer.
-	Execute(context.Context, *sorter) error
+	Execute(context.Context, *sorter, *engine.RocksDB, uint64) error
```

Comment:
This interface won't need to be changed if we construct the sorter with these, a strategy will access the member variables instead.